gravitee-apim-rest-api/gravitee-apim-rest-api-management/gravitee-apim-rest-api-management-rest/src/test/java/io/gravitee/rest/api/management/rest/spring/ResourceContextConfiguration.java

```diff
@@ -593,4 +597,14 @@ public ApiMetadataQueryService apiMetadataQueryService() {
     public ValidateApiDomainService validateApiDomainService() {
         return mock(ValidateApiDomainService.class);
     }
+
+    @Bean
```

Comment:
This should not be necessary. We should declare the in memory alternative only in https://github.com/gravitee-io/gravitee-api-management/blob/master/gravitee-apim-rest-api/gravitee-apim-rest-api-service/src/test/java/inmemory/spring/InMemoryConfiguration.java